gtk4/src/auto/inscription.rs

```diff
@@ -0,0 +1,968 @@
+    #[doc(alias = "gtk_inscription_new")]
+    pub fn new(text: Option<&str>) -> Inscription {
+        assert_initialized_main_thread!();
+        unsafe {
+            Widget::from_glib_none(ffi::gtk_inscription_new(text.to_glib_none().0)).unsafe_cast()
+        }
+    }
+
+    // rustdoc-stripper-ignore-next
+    /// Creates a new builder-pattern struct instance to construct [`Inscription`] objects.
+    ///
+    /// This method returns an instance of [`InscriptionBuilder`](crate::builders::InscriptionBuilder) which can be used to create [`Inscription`] objects.
+    pub fn builder() -> InscriptionBuilder {
+        InscriptionBuilder::default()
+    }
+
+    #[cfg(any(feature = "v4_8", feature = "dox"))]
+    #[cfg_attr(feature = "dox", doc(cfg(feature = "v4_8")))]
+    #[doc(alias = "gtk_inscription_get_attributes")]
+    #[doc(alias = "get_attributes")]
+    pub fn attributes(&self) -> Option<pango::AttrList> {
+        unsafe { from_glib_none(ffi::gtk_inscription_get_attributes(self.to_glib_none().0)) }
+    }
+
+    #[cfg(any(feature = "v4_8", feature = "dox"))]
+    #[cfg_attr(feature = "dox", doc(cfg(feature = "v4_8")))]
+    #[doc(alias = "gtk_inscription_get_min_chars")]
+    #[doc(alias = "get_min_chars")]
+    pub fn min_chars(&self) -> u32 {
+        unsafe { ffi::gtk_inscription_get_min_chars(self.to_glib_none().0) }
+    }
+
+    #[cfg(any(feature = "v4_8", feature = "dox"))]
+    #[cfg_attr(feature = "dox", doc(cfg(feature = "v4_8")))]
+    #[doc(alias = "gtk_inscription_get_min_lines")]
+    #[doc(alias = "get_min_lines")]
+    pub fn min_lines(&self) -> u32 {
+        unsafe { ffi::gtk_inscription_get_min_lines(self.to_glib_none().0) }
+    }
+
+    #[cfg(any(feature = "v4_8", feature = "dox"))]
+    #[cfg_attr(feature = "dox", doc(cfg(feature = "v4_8")))]
+    #[doc(alias = "gtk_inscription_get_nat_chars")]
+    #[doc(alias = "get_nat_chars")]
+    pub fn nat_chars(&self) -> u32 {
+        unsafe { ffi::gtk_inscription_get_nat_chars(self.to_glib_none().0) }
+    }
+
+    #[cfg(any(feature = "v4_8", feature = "dox"))]
+    #[cfg_attr(feature = "dox", doc(cfg(feature = "v4_8")))]
+    #[doc(alias = "gtk_inscription_get_nat_lines")]
+    #[doc(alias = "get_nat_lines")]
+    pub fn nat_lines(&self) -> u32 {
+        unsafe { ffi::gtk_inscription_get_nat_lines(self.to_glib_none().0) }
+    }
+
+    #[cfg(any(feature = "v4_8", feature = "dox"))]
+    #[cfg_attr(feature = "dox", doc(cfg(feature = "v4_8")))]
+    #[doc(alias = "gtk_inscription_get_text")]
+    #[doc(alias = "get_text")]
+    pub fn text(&self) -> Option<glib::GString> {
+        unsafe { from_glib_none(ffi::gtk_inscription_get_text(self.to_glib_none().0)) }
+    }
+
+    //#[cfg(any(feature = "v4_8", feature = "dox"))]
+    //#[cfg_attr(feature = "dox", doc(cfg(feature = "v4_8")))]
+    //#[doc(alias = "gtk_inscription_get_text_overflow")]
+    //#[doc(alias = "get_text_overflow")]
+    //pub fn text_overflow(&self) -> /*Ignored*/InscriptionOverflow {
```

Comment:
InscriptionOverflow needs to be added to Gir.toml

Reply:
Indeed good catch, I missed this.